### PR TITLE
Add documentation for LDAP otions added in PR #291

### PR DESCRIPTION
**Pull request by Primer42**
@tumblr/systems

### CODE REVIEW

File: _includes/configuration/authentication.html [hunk 1]
```diff
@@ -111,41 +111,73 @@
...
       <td class="inlinecode">usersub</td>
       <td>String</td>
-      <td>Where to find users in the LDAP schema</td>
+      <td>Where to find users in the LDAP schema.  This is a required configuration.</td>
     </tr>
     <tr>
       <td class="inlinecode">groupsub</td>
       <td>String</td>
-      <td>Where to find groups in the LDAP schema</td>
+      <td>Where to find groups in the LDAP schema. This is a require configuration.</td>
```
Comment (defect) [inline _includes/configuration/authentication.html]: s/require/required/

File: _includes/configuration/authentication.html [hunk 2]
```diff
@@ -111,41 +111,73 @@
...
+    </tr>
+    <tr>
+        <td class="inlinecode">userNumberAttribute</td>
+        <td>String</td>
+        <td>The LDAP schema attribute used to identify the user id number. Defaults to <span class="inlinecode">uidNumber</span></td>
+    </tr>
+    <tr>
+        <td class="inlinecode">groupAttribute</td>
+        <td>String</td>
+        <td>An LDAP schema attribute to use for retrieving group information.  This is a require configuration</td>
```
Comment (defect) [inline _includes/configuration/authentication.html]: s/require/required/

File: _includes/configuration/authentication.html (hunk 1 above)
Comment (Primer42) [inline _includes/configuration/authentication.html]: Fixed

File: _includes/configuration/authentication.html (hunk 2 above)
Comment (Primer42) [inline _includes/configuration/authentication.html]: Fixed

**Pull request merged**